driver-core/src/test/unit/com/mongodb/internal/connection/ScramShaAuthenticatorSpecification.groovy

```diff
@@ -60,10 +60,10 @@ class ScramShaAuthenticatorSpecification extends Specification {
         def authenticator = new ScramShaAuthenticator(credential, { 'fyko+d2lbbFgONRv9qkxdawL' }, { preppedPassword })
 
         then:
-        validateAuthentication(payloads, authenticator, async)
+        validateAuthentication(payloads, authenticator, async as boolean, emptyExchange as boolean)
```

Comment:
Sorry, missed this last review... Why all the `as boolean`?  

If you agree they are unnecessary, please remove them.

Reply:
The error highlighting in IntelliJ was annoying me, so I added the casts. I have removed them.